SonarQube.Common/AnalysisConfig/AnalysisSetting.cs

```diff
@@ -70,26 +70,7 @@ public static bool IsValidKey(string key)
             bool isValid = ValidSettingKeyRegEx.IsMatch(key);
             return isValid;
         }
-
-        /// <summary>
```

Comment:
The ArgumentProcessor not longer does the cmd line arg processing of /p: arguments, so this method is now redundant.